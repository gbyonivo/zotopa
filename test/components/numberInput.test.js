import React from 'react';
import renderer from 'react-test-renderer';
import NumberInput from '../../src/components/textInput';

describe('NumberInput component', () => {
  it('TextInput renders correctly', () => {
    const actual = renderer.create(
      <NumberInput
        value={0}
        label="play"
        name="play"
        placeholder="play"
        onChange={() => {}}
      />
    );
    expect(actual).toMatchSnapshot();
  });
});
