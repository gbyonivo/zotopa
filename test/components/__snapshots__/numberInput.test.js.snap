// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumberInput component TextInput renders correctly 1`] = `
<div
  className="sc-bdVaJa lhoMHm"
>
  <label
    className="sc-bwzfXH gCFQuH"
    htmlFor="play"
  >
    play
  </label>
  <input
    className="sc-htpNat iCiSUl"
    name="play"
    onChange={[Function]}
    placeholder="play"
    type="text"
    value={0}
  />
</div>
`;
